include/bbp/sonata/optional.hpp

```diff
@@ -0,0 +1,1715 @@
+//
```

Comment:
Such pain. Should this file not be vendored properly?

Reply:
how? I can add this to spack but **it** (totally undefined) want to limit number of depends.